packages/model/src/reminder.ts

```diff
@@ -2,11 +2,15 @@ import { ListResponse } from './common';
 import { EventResponse } from './event';
 import { ResearchOutputDataObject } from './research-output';
 
-type ReminderEntity = 'Research Output' | 'Event';
+type ReminderEntity = 'Research Output' | 'Event' | 'Event Material';
```

Comment:
I think the point of this union is to list all entities (ie Research Output, Event, User) that the reminder might be relating to. `Event Material` really isn't one anyway and I think it should just be `Event`